apps/hashdotai/docs/hash/working-with-data-in-pages/using-blocks-in-pages.md

```diff
@@ -0,0 +1,68 @@
+
+## How Blocks work with Entities, Links, and Entity Types {#how-blocks-work-with-entities-links-and-entity-types}
+
+Blocks can render and manipulate Entities, Links, Entity Types, and even Property, Data, and Link Types, though this is less common. Blocks can also create new Entities, Links, and Entity Types.
+
+You might use a Block which renders some Entities and Links which already exist in your HASH Workspace. Or you might use a Block to create new Entities of an Entity Type which doesn’t exist yet. Since Blocks are developed by third-parties, there is a huge range of possible data workflows which they enable.
+
+## The default Block: Paragraph Block {#the-default-block-paragraph-block}
+
+When you create a new Page in HASH, a Paragraph Block will automatically be inserted at the top of the page and your cursor will be placed inside the block, ready to type. HASH does this so that Pages can be used to quickly capture some writing.
+
+The Paragraph Block creates a Text Entity in your HASH Workspace. The Text Entity conforms to the Text Entity Type, which is a non-editable system type. As you write in the Paragraph Block, the ‘Tokens’ Property on the Text Entity is updated to capture your text.
+
+Hitting the return key in the Paragraph Block will create a new Paragraph Block below the first one. Each Paragraph on a Page will be its own Block. You can tell this from the [...] context menu on the left of the block, from the comment button on the right of the block, and from the —[+]— button which appears above and below the Paragraph when you hover.
+
+![alt_text](images/image5.png "image_tooltip")
+
+## The Block context menu {#the-block-context-menu}
+
+The [...] button on the left of the block is its context menu, providing a set of options:
+
+1.  **Load block from URL**: this is for block developers to test their Blocks.
+1.  **Copy Link**: this copies a link to this specific block to your clipboard. Opening this link will open the Page and show a visual highlight next to the block which was linked to.
+1.  **Edit Block**: this opens the Entity Editor for the Entity attached to this Block in a side-bar overlay on the right of the screen. Edits you make to the Entity will be reflected in the Block immediately.
+1.  **Configure**: this opens the block’s configuration menu. Not all blocks have configuration options. Those that do allow for things like the customization of the block’s UI or the toggling of different features.
+1.  **Delete**: deletes the block.
+
+## Inserting a Block {#inserting-a-block}
+
+The easiest way to insert a new block is to type a “/”. This will open a menu of all the different types of Blocks available in HASH. You can scroll through this list or continue typing to search. Hitting return will insert the currently selected block in that menu, but you can also click on the block you want. The “/” command works in the Paragraph, Heading, and Callout blocks.
```

Comment:
```suggestion
The easiest way to insert a new block is to press the `/` key on your keyboard. This will open a menu of all the different types of blocks available in HASH. You can scroll through this list or continue typing to search. Hitting return will insert the currently selected block in that menu, but you can also click on the block you want. The `/` command works in almost any text-based block.
```